packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -175,6 +193,18 @@ export default defineComponent<{
 				})
 				return
 			}
+
+			if (props.column?.type === Kotti.Filters.FilterType.INTEGER) {
+				emit('input', {
+					...newFilter,
+					value:
+						typeof newFilter.value === 'number'
+							? Math.trunc(newFilter.value)
```

Comment:
Yes, even with the `step` set to 1, I can still type numbers with decimals and the values are saved with decimals

Reply:
will resolve since we thoroughly discussed this issue to be on `KtFieldNumber` not properly handling the `isStepMultiple` validator

issue will be resolved independantly